server/modules/strelka/strelka.go

```diff
@@ -361,6 +374,10 @@ func (e *StrelkaEngine) startCommunityRuleImport() {
 
 				for _, rule := range parsed {
 					det := rule.ToDetection(repo.License, filepath.Base(repopath))
+					log.WithFields(log.Fields{
+						"Id":    det.PublicID,
```

Comment:
These log fields need to be more specific to avoid elastic type collisions with other log lines in the product. Perhaps `yaraRuleId` and `yaraRuleTitle`. Use the same field names with the similar log lines also added in this PR.